### DISCUSSION TITLE
ESP CAM - Browser - UTF 8 Problem

### DISCUSSION THREAD

**Discussion by spitfire4all** [easytarget/esp32-cam-webserver#275] [Common Issues]
Hi,
maybe someone can help me with an old projekt with esp cam.

After one year I check my camera again but I see no streams and Browser shows a Websocket utf 8 problem.
No reaktion from the buttons and no stream was shown.

![image](https://user-images.githubusercontent.com/25489051/201946516-2f1a7f17-9ab4-4e7e-98c5-7401d503647c.png)

![image](https://user-images.githubusercontent.com/25489051/201946914-a009564c-2f2c-4fbd-8ceb-bf8bed4d1bad.png)

Regards Dieter

**Comment by easytarget** [comment]
Sorry; I am not doing any further development on this project. it is old code and somewhat obsolete. I am currently working on retiring it; I will try to link to a more actively updated fork when I do.
